src/Tizen.NUI.Scene3D/src/ModelView.cs

```diff
@@ -76,58 +70,74 @@ public ModelView(ModelView modelView) : this(Interop.ModelView.NewModelView(Mode
         }
 
         /// <summary>
-        /// Assignment operator.
+        /// Retrieves model root View.
         /// </summary>
-        /// <param name="modelView">Handle to an object.</param>
-        /// <returns>Reference to this.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public ModelView Assign(ModelView modelView)
+        public View ModelRoot
         {
-            ModelView ret = new ModelView(Interop.ModelView.ModelViewAssign(SwigCPtr, ModelView.getCPtr(modelView)), false);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            get
+            {
+                View ret = new View(Interop.ModelView.GetModelRoot(SwigCPtr), false);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return ret;
+            }
         }
 
         /// <summary>
-        /// Retrieves model root Actor.
+        /// Fits the model to the Control size.
         /// </summary>
-        /// <returns>Root View of the model.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public View GetModelRoot()
+        public bool FitSize
         {
-            View ret = new View(Interop.ModelView.GetModelRoot(SwigCPtr), false);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            get
+            {
+                return fitSize;
+            }
+            set
+            {
+                fitSize = value;
+                Interop.ModelView.FitSize(SwigCPtr, fitSize);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
         }
 
         /// <summary>
-        /// Fits the model to the Control size.
+        /// Moves the model to the center of control.
         /// </summary>
-        /// <param name="fit">True to fit model size to control.</param>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public void FitSize(bool fit)
+        public bool FitCenter
```

Comment:
is it possible to use FitSize and FitCenter both? is there any differences by using FitSize only?

Reply:
Thank you I added more comments.